plugins/aws/iam/rootAccountInUse.js

```diff
@@ -75,11 +79,11 @@ module.exports = {
 					helpers.addResult(results, 0, 'Root account has not been used', 'global', obj.arn);
 				} else {
 					var dateToCompare = helpers.functions.mostRecentDate(accessDates);
-					var resultCode = (helpers.functions.daysAgo(dateToCompare) < config.root_account_in_use_days) ? 2: 0;
+					var resultCode = (helpers.functions.daysBetween(dateToCompare, now) < config.root_account_in_use_days) ? 2: 0;
```

Comment:
Functionally this looks similar, but why was this changed to `daysBetween`?

Reply:
For testing. The function "daysAgo" always gets the current date/time. In order to test this, I needed a way to inject the current time so that I could test different scenarios. This approach allowed me to inject `now` into function for testing.